query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/simple/find_first.rs

```diff
@@ -35,6 +35,19 @@ mod find_first_query {
         Ok(())
     }
 
+    #[connector_test]
```

Comment:
Added a _failure_ case for `findFirst` to check that it actually behaves differently than `findFirstOrThrow`